src/main/java/io/anserini/index/IndexUtils.java

```diff
@@ -138,6 +183,51 @@ public void printDocumentVector(String docid) throws IOException, NotStoredExcep
     }
   }
 
+  public void getAllDocids(Compression compression) throws IOException {
+    Query q = new FieldValueQuery(LuceneDocumentGenerator.FIELD_ID);
+    IndexSearcher searcher = new IndexSearcher(reader);
+    ScoreDoc[] scoreDocs;
+    try {
+      scoreDocs = searcher.search(new FieldValueQuery(LuceneDocumentGenerator.FIELD_ID), reader.maxDoc(),
+              BREAK_SCORE_TIES_BY_DOCID).scoreDocs;
```

Comment:
I think we want 4-space indent here?

Reply:
You mean the multi-lines statement?
I just use my IntelliJ's default indent....and it seems that it doesn't allow me to have self-defined indent here.... (backspace will go to the line above, enter will do the exactly same here)

Reply:
I believe it's called a "continuation indent" in IntelliJ... set to four spaces?